### PR TITLE
Enable dynamic notch warning

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: locales/en/messages.json
Comment (KarateBrot) [inline locales/en/messages.json:4030]:
```suggestion
    "pidTuningDynamicNotchFilterNyquistWarning": {
```

File: src/tabs/pid_tuning.html
Comment (KarateBrot) [inline src/tabs/pid_tuning.html:1522]:
```suggestion
                                        <p i18n="pidTuningDynamicNotchFilterNyquistWarning"></p>
```

**Review by KarateBrot — CHANGES_REQUESTED** [review]
This is more of a suggestion to give it a shorter name

File: locales/en/messages.json
Comment (haslinghuis) [reply locales/en/messages.json]: Done.

File: src/tabs/pid_tuning.html
Comment (haslinghuis) [reply src/tabs/pid_tuning.html]: Done.

**Comment by haslinghuis** [comment]
image

**Comment by haslinghuis** [comment]
After using a preset and arriving on the page we should adjust the message.
Otherwise we should enforce in firmware itself.

**Comment by haslinghuis** [comment]
We are using tables in this tab for presentation. Tried margin but not getting expected results. Already spent too much time on presentation as we need to migrate to templates.

image

**Review by KarateBrot — APPROVED** [review]

**Comment by KarateBrot** [comment]
> We are using tables in this tab for presentation. Tried margin but not getting expected results. Already spent too much time on presentation as we need to migrate to templates.

Looks pretty good anyway! 👌

**Review by McGiverGim — APPROVED** [review]

**Pull request merged**
